common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -290,6 +299,91 @@ def _check_scores_and_page_text(self, problem_scores, section_score, text):
         self.assertEqual(self._get_section_score(), section_score)
         self.assertTrue(self.progress_page.text_on_page(text))
 
+    def _check_tick_text(self, index, sr_text, label, label_hidden=True):
+        """
+        Check the label and sr text for a horizontal (X-axis) tick.
+        """
+        self.assertEqual(sr_text, self.progress_page.x_tick_sr_text(index))
+        self.assertEqual([label, 'true' if label_hidden else None], self.progress_page.x_tick_label(index))
+
+    def test_axis_a11y(self):
+        """
+        Tests that the progress chart axis have appropriate a11y (screenreader) markup.
+        """
+        with self._logged_in_session():
+            self.courseware_page.visit()
+            self._answer_problem_correctly()
```

Comment:
Nit: add comments about what is being answered (first time, the HW problem, second time the Lab problem).